samples/kubernetes/guestbook/deploy/frontend-deployment.yaml

```diff
@@ -16,7 +16,7 @@ spec:
     spec:
       containers:
       - name: php-redis
-        image: gcr.io/google-samples/gb-frontend:v4
+        image: ghcr.io/radius-project/gb-frontend:v4  
```

Comment:
should we should follow the same registry path as demo and eshop samples `ghcr.io/radius-project/samples/<sampleapp>`?

Reply:
good catch! updated.